iroh-base/src/hash.rs

```diff
@@ -360,6 +360,21 @@ mod tests {
         assert_eq!(ser.len(), 32);
     }
 
+    #[test]
+    #[ignore]
+    // TODO(ramfox): this was used in an example, but it seems to always return a `DeserialzeUnexpectedEnd` postcard error
+    // should this be expected to work
+    fn test_hash_list_postcard() {
+        let hash1 = Hash::new("hello");
+        let hash2 = Hash::new("world");
+        let mut ser = postcard::to_stdvec(&hash1).unwrap();
+        let mut ser2 = postcard::to_stdvec(&hash2).unwrap();
+        ser.append(&mut ser2);
+        let hashes: Box<[Hash]> = postcard::from_bytes(&ser).unwrap();
+        assert_eq!(hash1, hashes[1]);
+        assert_eq!(hash2, hashes[2]);
+    }
+
```

Comment:
`let hashes: Box<[Hash]> = postcard::from_bytes(&ser).unwrap();` was taken from the previous `dump-blob-stream` example code, but it always failed when I wrote the `fetch-fsm` and `fetch-stream` examples. 

Should we be expecting this to work?

Reply:
cc @rklaehn

Reply:
How can this ever have worked? A Box<[Hash]> would be a varint encoded "2" and then 2 hashes, not just 2 concatenated hashes. I have no idea what this was supposed to test, so I would just drop it.